### PR TITLE
XSIAM-VMware-vCenter-Modeling-Enhancement-CIAC-7533

### PR DESCRIPTION
**Pull request by cweltPA**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-7533](https://jira-hq.paloaltonetworks.local/browse/CIAC-7533)

## Description
This pull request carries XSIAM modeling rules enhancements for the VMware vCenter pack.
It adds support for mapping dozens of additional events. 

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by eepstain — APPROVED** [review]

File: Packs/VMwareVCenter/README.md [hunk 1]
```diff
@@ -1,8 +1,40 @@
...
+This pack includes Cortex XSIAM content.
+  
+## Configuration on Server Side
+This section describes the configuration required on the VMware vCenter server to forward its event logs to Cortex XSIAM Broker VM via syslog.
+ 
+1. Log in to the vCenter Server Management Interface as user root.
+2. In the vCenter Server Management Interface, select **Syslog**.
+3. In the *Forwarding Configuration* section - 
+   * If you have not yet configured any remote syslog hosts, click **Configure**. 
+   * If you already have configured hosts previously, Click **Edit**.
```
Comment (ShirleyDenkberg) [inline Packs/VMwareVCenter/README.md:15]:
```suggestion
   * If you configured hosts previously, click **Edit**.
```

File: Packs/VMwareVCenter/README.md [hunk 2]
```diff
@@ -1,8 +1,40 @@
...
-2. Right-click, and select **Syslog Collector** -> **Configure**.
-3. When configuring the Syslog Collector, set:
-   - vendor as vendor<-  VMware
-   - product as product<- VCenter
+1. Navigate to **Settings** &rarr; **Configuration** &rarr; **Data Broker** &rarr; **Broker VMs**. 
+2. Right-click, and select **Syslog Collector** &rarr; **Configure**.
+3. When configuring the Syslog Collector, set the following parameters:
+   | Parameter     | Value    
+   | :---          | :---                    
+   | `Protocol`    | Should be aligned with the selected *protocol* value in the vCenter Server Management Interface syslog configuration, as described on the [Configuration on Server Side](#configuration-on-server-side) above.
```
Comment (ShirleyDenkberg) [inline Packs/VMwareVCenter/README.md:34]:
```suggestion
   | `Protocol`    | Should be aligned with the selected *protocol* value in the vCenter Server Management Interface syslog configuration, as described in the [Configuration on Server Side](#configuration-on-server-side) section above.   
```

File: Packs/VMwareVCenter/README.md [hunk 3]
```diff
@@ -1,8 +1,40 @@
...
-3. When configuring the Syslog Collector, set:
-   - vendor as vendor<-  VMware
-   - product as product<- VCenter
+1. Navigate to **Settings** &rarr; **Configuration** &rarr; **Data Broker** &rarr; **Broker VMs**. 
+2. Right-click, and select **Syslog Collector** &rarr; **Configure**.
+3. When configuring the Syslog Collector, set the following parameters:
+   | Parameter     | Value    
+   | :---          | :---                    
+   | `Protocol`    | Should be aligned with the selected *protocol* value in the vCenter Server Management Interface syslog configuration, as described on the [Configuration on Server Side](#configuration-on-server-side) above.   
+   | `Port`        | Should be aligned with the *port* defined in the vCenter Server Management Interface syslog configuration as described on the [Configuration on Server Side](#configuration-on-server-side) above.
```
Comment (ShirleyDenkberg) [inline Packs/VMwareVCenter/README.md:35]:
```suggestion
   | `Port`        | Should be aligned with the *port* defined in the vCenter Server Management Interface syslog configuration as described in the [Configuration on Server Side](#configuration-on-server-side) section above.   
```

File: Packs/VMwareVCenter/README.md [hunk 4]
```diff
@@ -1,8 +1,40 @@
...
-   - product as product<- VCenter
+1. Navigate to **Settings** &rarr; **Configuration** &rarr; **Data Broker** &rarr; **Broker VMs**. 
+2. Right-click, and select **Syslog Collector** &rarr; **Configure**.
+3. When configuring the Syslog Collector, set the following parameters:
+   | Parameter     | Value    
+   | :---          | :---                    
+   | `Protocol`    | Should be aligned with the selected *protocol* value in the vCenter Server Management Interface syslog configuration, as described on the [Configuration on Server Side](#configuration-on-server-side) above.   
+   | `Port`        | Should be aligned with the *port* defined in the vCenter Server Management Interface syslog configuration as described on the [Configuration on Server Side](#configuration-on-server-side) above.   
+   | `Format`      | Select **Auto-Detect**. 
+   | `Vendor`      | Enter *VMware*.
```
Comment (ShirleyDenkberg) [inline Packs/VMwareVCenter/README.md:37]:
```suggestion
   | `Vendor`      | Enter **VMware**.
```

File: Packs/VMwareVCenter/README.md [hunk 5]
```diff
@@ -1,8 +1,40 @@
...
+   | `Product`     | Enter *vCenter*.
```
Comment (ShirleyDenkberg) [inline Packs/VMwareVCenter/README.md:38]:
```suggestion
   | `Product`     | Enter **vCenter**.
```

File: Packs/VMwareVCenter/README.md [hunk 6]
```diff
@@ -1,8 +1,40 @@
...
+2. In the vCenter Server Management Interface, select **Syslog**.
+3. In the *Forwarding Configuration* section - 
+   * If you have not yet configured any remote syslog hosts, click **Configure**. 
+   * If you already have configured hosts previously, Click **Edit**.
+4. From the **Protocol** drop-down menu, select the requested protocol for the syslog forwarding (for example *UDP*).
+5. In the **Port** text box, enter the port number that would be used for communication with the target Cortex XSIAM Broker VM syslog server.
+6. In the *Create Forwarding Configuration* pane, click **Add** to enter another remote syslog server.
+7. Click **Save**.
+ 
+For additional details, see [Forward vCenter Server Log Files to Remote Syslog Server](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.monitoring.doc/GUID-9633A961-A5C3-4658-B099-B81E0512DC21.html)]
```
Comment (ShirleyDenkberg) [inline Packs/VMwareVCenter/README.md:21]:
```suggestion
For additional details, see [Forward vCenter Server Log Files to Remote Syslog Server](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.monitoring.doc/GUID-9633A961-A5C3-4658-B099-B81E0512DC21.html)
```

**Comment by ShirleyDenkberg** [comment]
@eepstain @evisochek Doc review completed.

**Pull request merged**